build/build-image/Dockerfile

```diff
@@ -53,7 +53,7 @@ RUN go install golang.org/x/tools/cmd/goimports@latest && \
 #
 # When this is raised to 1.29: bump the code-generator version on line 66 to the v0.30 tag.
 # When this is raised to >=1.30: delete line 66-67 and uncomment line 68 to stop pinning code-generator.
-ENV KUBERNETES_VER 1.29.7
+ENV KUBERNETES_VER 1.31.0
```

Comment:
Kubernetes cluster version should be the middle version, this this case K8s 1.30. The test cluster is currently at patch version 1.30.5 so we should use that one.

Reply:
Interesting, for some reason `https://storage.googleapis.com/kubernetes-release/release/v1.30.5/bin/linux/amd64/kubectl` does not exist. If we use 1.30.4 instead of 1.30.5 for both Kubernetes and Kubectl versions it should work.